pepper-apis/src/main/java/org/broadinstitute/ddp/cache/CacheService.java

```diff
@@ -68,12 +69,13 @@ private CacheService() {
             }
             URI redissonConfigPathUri = redissonConfigPath.toUri();
             cacheManager = buildCacheManager(redissonConfigPathUri);
-            String redisAddress = ConfigManager.getInstance().getConfig().getString(ConfigFile.REDIS_SERVER_ADDRESS);
-            Config redissonConfig = new Config();
-            redissonConfig.useSingleServer()
-                    .setTimeout(30000)
```

Comment:
This is where the 30s timeouts where coming from. We were also picking up all the default Redisson configuration settings for the local caches instead of the tweaked settings we were trying out to get past the errors.

Reply:
Nice!